src/components/comments/CommentsPanelForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useRef } from 'react';
+import { callAPI } from '../../actions/WindowActions';
+import { updateCommentsPanelTextInput } from '../../actions/CommentsPanelActions';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { translateCaption } from '../../utils/index';
+
+export const CommentsPanelForm = (props) => {
+  const { textInput, postComment, updateText, docId, windowId } = props;
+  const textRef = useRef(null);
+  const sendButton = translateCaption('window.comments.add');
+
+  /**
+   * Focus by default on the text input
+   */
+  useEffect(() => {
+    textRef.current.focus();
+  }, []);
+
+  return (
+    <div className="col-12">
```

Comment:
If this is finally a valid form, why not use the `form` element ?

Reply:
changed